### PR TITLE
[blog] set max image width

### PR DESCRIPTION
**Pull request by chejennifer**

### CODE REVIEW

File: _sass/blog.scss [hunk 1]
```diff
@@ -15,4 +15,8 @@
   color: var(--dc-gray);
   text-transform: uppercase;
   font-size: 1rem;
+}
+
+img {
```
Comment (beets) [inline _sass/blog.scss:20]: Can we restrict this to blog pages? There are other images with text (e.g. tutorials) which would be too small to read if we restrict the width.

**Review by beets — COMMENTED** [review]
Also, sorry for not catching this earlier, but could you also update the blog post image to be from a smaller window width and height.

File: _sass/blog.scss (hunk 1 above)
Comment (chejennifer) [reply _sass/blog.scss]: done

**Comment by chejennifer** [comment]
> Also, sorry for not catching this earlier, but could you also update the blog post image to be from a smaller window width and height.

what do you mean by updating the image to be from a smaller window width and height? like take a new screenshot? what size are you thinking

**Comment by beets** [comment]
> > Also, sorry for not catching this earlier, but could you also update the blog post image to be from a smaller window width and height.
> 
> Do you mean take a new screenshot for the blogpost? what size are you thinking

Yup - new screenshot. Maybe 960 (w/o stat var panel) or 1080 w/ stat var panel

**Comment by chejennifer** [comment]
> > > Also, sorry for not catching this earlier, but could you also update the blog post image to be from a smaller window width and height.
> > 
> > 
> > Do you mean take a new screenshot for the blogpost? what size are you thinking
> 
> Yup - new screenshot. Maybe 960 (w/o stat var panel) or 1080 w/ stat var panel

updated

**Review by beets — APPROVED** [review]
thx!

**Pull request merged**